src/components/Items/ProfileContact.js

```diff
@@ -68,7 +68,7 @@ const ProfileContact = ({
           )}
         {select && (
           <View testID="selected">
-            <MemoIc_Checklist />
+            <IconChecklist />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The usage of the icon component has been updated to reflect the new name `IconChecklist`. This change is consistent with the updated import statement.
```diff
- <MemoIc_Checklist />
+ <IconChecklist />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->